eZ/Publish/API/Repository/Mapper/SortClauseMapper.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * File containing SortClauseMapper class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository\Mapper;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+
+interface SortClauseMapper
```

Comment:
This one is not needed, Mapper is not part of the API, it's an internal service.